kyo-core/shared/src/main/scala/kyo/ios.scala

```diff
@@ -1,25 +1,28 @@
 package kyo
 
+import IOs.internal.*
 import core.*
-import core.internal.*
-import iosInternal.*
 import java.util.concurrent.atomic.AtomicReference
 import scala.annotation.tailrec
 import scala.util.*
 import scala.util.control.NonFatal
 
-sealed trait IOs extends Effect[IOs]:
+opaque type IOs <: Defers = Defers & SideEffects
```

Comment:
Because this is a subtype of `Defers`, I think this should be possible: 

`Defers.run(IOs(sideEffect)).pure`

Will this work?
`opaque type IOs >: Defers = Defers & SideEffects`